pkg/support/dump.go

```diff
@@ -293,7 +293,14 @@ func (d *agentDumper) DumpAgentInfo(basedir string) error {
 }
 
 func (d *agentDumper) DumpMemberlist(basedir string) error {
-	return dumpAntctlGet(d.fs, d.executor, "memberlist", basedir)
+	output, err := d.executor.Command("antctl", "-oyaml", "get", "memberlist").CombinedOutput()
+	if err != nil {
+		if strings.Contains(string(output[:]), "memberlist is not running") {
+			return writeFile(d.fs, filepath.Join(basedir, "memberlist"), "memberlist", nil)
```

Comment:
I think we should write the output to the file to let the bundle debugger know this information (memberlist is not running). So the code could just be:
```
if err != nil && !strings.Contains(string(output), "memberlist is not running") {
    return fmt.Errorf("error when dumping memberlist: %w", err)
}
return writeFile(d.fs, filepath.Join(basedir, "memberlist"), "memberlist", output)
```